custom/logistics/tasks.py

```diff
@@ -41,10 +42,11 @@ def stock_data_task(domain, endpoint, apis, test_facilities=None):
     if TEST:
         facilities = test_facilities
     else:
-        facilities = SQLLocation.objects.filter(
+        supply_points_ids = SQLLocation.objects.filter(
             domain=domain,
-            location_type__iexact='FACILITY'
-        ).order_by('created_at').values_list('external_id', flat=True)
+            location_type__in=get_reporting_types(domain)
+        ).order_by('created_at').values_list('supply_point_id', flat=True)
+        facilities = [doc['external_id'] for doc in iter_docs(SupplyPointCase.get_db(), supply_points_ids)]
```

Comment:
@czue Not related with your changes but in EWS supply_point_id != location_id so we need fetch external_id of supply point to correctly migrate data.